content/02.main-text.md

```diff
@@ -432,7 +432,15 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Concepts for the future of scholarly publishing extend beyond collaborative writing [@doi:10.22541/au.148769949.92783646; @doi:10.22541/au.149693987.70506124; @tag:techblog-brown].
 Pandoc Scholar [@doi:10.7717/peerj-cs.112] and Bookdown [@doi:10.1201/9781315204963], which has been used for open writing [@doi:10.12688/f1000research.16516.1], both enhance traditional Markdown to better support publishing.
-The [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+[Dokieli](https://dokie.li/) is a clientside editor for HTML articles, which aims for decentralization by adhering to web standards that allow articles and reader annotations to be stored by any participating server [@url:https://csarven.ca/dokieli-rww].
+[Texture](https://substance.io/texture/) is also browser-based editor, but uses JATS as the primary storage format.
```

Comment:
@dhimmel I didn't notice until after merging, but this should be `also *a* browser-based`. Should one of us quickly fix it on `master`?

Reply:
done in https://github.com/greenelab/meta-review/commit/51821359ee2872aacfa5fe9d8ce0b102c5d91396